src/utils/faceting-utils.ts

```diff
@@ -45,3 +47,115 @@ export function getInitialFacetPanelOpen(config: RecordsetConfig, initialReferen
   }
   return res;
 }
+
+export const getFacetOrderStorageKey = (reference: any): string => {
+  return `facet-order-${reference.table.schema.catalog.id}_${reference.table.schema.name}_${reference.table.name}`;
```

Comment:
Do we want to have the same configuration for this table when it's shown in different contexts? I see that this is the specified "key" to use in the issue, but wasn't sure if that was discussed.

Reply:
Yeah, we want this to be used in different contexts. I'll comment this